jni/nnstreamer.mk

```diff
@@ -67,7 +67,8 @@ NNSTREAMER_PLUGINS_SRCS := \
     $(NNSTREAMER_GST_HOME)/elements/gsttensor_sparseutil.c \
     $(NNSTREAMER_GST_HOME)/elements/gsttensor_split.c \
     $(NNSTREAMER_GST_HOME)/elements/gsttensor_transform.c \
-    $(NNSTREAMER_GST_HOME)/tensor_filter/tensor_filter.c
+    $(NNSTREAMER_GST_HOME)/tensor_filter/tensor_filter.c \
+    $(NNSTREAMER_GST_HOME)/tensor_train/gsttensor_trainsink.c
```

Comment:
How about adding new flag for training elements?
```
NNSTREAMER_TRAINING_SRCS := \
    $(NNSTREAMER_GST_HOME)/tensor_train/gsttensor_trainsink.c \
    $(NNSTREAMER_GST_HOME)/tensor_train/gsttensor_train....
```

Reply:
Thank you.